auth/store.go

```diff
@@ -851,8 +851,9 @@ func (as *authStore) isOpPermitted(userName string, revision uint64, key, rangeE
 	if revision == 0 {
 		return ErrUserEmpty
 	}
-
-	if revision < as.Revision() {
+	rev := as.Revision()
+	if revision < rev {
+		as.lg.Warn("isOppermitted", zap.Uint64("auth store revision", rev), zap.Uint64("request auth revision", revision), zap.ByteString("request key", key), zap.Error(ErrAuthOldRevision))
```

Comment:
Change the error message "isOppermitted" to something more relevant? Maybe "operation on key is not permitted". The logger already prints line number, so we know it is from function isOpPermitted. Also, I think multiline is preferred for better readability.

Reply:
thanks. updated. PTAL.